src/dev/performance/run_performance_cli.ts

```diff
@@ -6,14 +6,74 @@
  * Side Public License, v 1.
  */
 
+import { createFlagError } from '@kbn/dev-cli-errors';
 import { run } from '@kbn/dev-cli-runner';
 import { REPO_ROOT } from '@kbn/utils';
-import Fsp from 'fs/promises';
+import fs from 'fs';
 import path from 'path';
 
+const JOURNEY_BASE_PATH = 'x-pack/performance/journeys';
+
+export interface Journey {
+  name: string;
+  path: string;
+}
```

Comment:
I'm not sure if it is a good idea overall, just for the logs it is easy to read file name instead of path.
I defined it here and import in `run_scalability_cli` for consistency.